Tangram-es.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
 
   s.name    = 'Tangram-es'
-  s.version = '0.5.1'
+  s.version = '0.6.0'
```

Comment:
Just updated the docs on tangram-docs, `s.documentation_url` can be updated to point to `https://mapzen.com/documentation/tangram/ios-framework/0.6.0/`. Let's hold on merging until https://github.com/tangrams/tangram-docs/pull/208 is in.